tests/autologging/test_autologging_behaviors_integration.py

```diff
@@ -90,14 +90,26 @@ def test_autologging_integrations_use_safe_patch_for_monkey_patching(integration
         ) as gorilla_mock, mock.patch(
             integration.__name__ + ".safe_patch", wraps=safe_patch
         ) as safe_patch_mock:
-            integration.autolog(disable=False)
-            assert safe_patch_mock.call_count > 0
+            if integration.__name__ == "mlflow.xgboost":
```

Comment:
Can we insert an inline comment explaining the difference in logic here?

Reply:
@dbczumar Since we enable `mlflow.sklearn._autolog()` inside `mlflow.xgboost`, sklearn methods are patched to XGBoost sklearn models. Their patch call should be counted as well. I will add a comment to explain the code difference. Thank you for your suggestion!